dojo/models.py

```diff
@@ -688,12 +688,29 @@ def findings_count(self):
             # ideally it's always prefetched and we can remove this code in the future
             self.active_finding_count = Finding.objects.filter(mitigated__isnull=True,
                                             active=True,
+                                            verified=False,
                                             false_p=False,
                                             duplicate=False,
                                             out_of_scope=False,
                                             test__engagement__product=self).count()
             return self.active_finding_count
 
+    @cached_property
+    def findings_active_verified_count(self):
+        try:
+            # if prefetched, it's already there
+            return self.active_verified_finding_count
+        except AttributeError:
+            # ideally it's always prefetched and we can remove this code in the future
+            self.active_verified_finding_count = Finding.objects.filter(mitigated__isnull=True,
+                                            active=True,
+                                            verified=True,
+                                            false_p=False,
+                                            duplicate=False,
+                                            out_of_scope=False,
```

Comment:
same

Reply:
No, not that one. That one is actually to filter specifically `verified`.